src/Sarif.UnitTests/Writers/ResultLogJsonWriterTests.cs

```diff
@@ -13,14 +14,17 @@ namespace Microsoft.CodeAnalysis.Sarif.Writers
     [TestClass]
     public class ResultLogJsonWriterTests
     {
+        private static readonly string SchemaVersion =
+            SarifUtilities.ConvertToText(SarifVersion.OneZeroZeroBetaFour);
+
```

Comment:
This avoids having to change every test in the file when the schema version changes.